tests/python/pants_test/backend/project_info/tasks/test_filedeps.py

```diff
@@ -26,7 +26,7 @@ def task_type(cls):
     return FileDeps
 
   def setUp(self):
-    super(FileDepsTest, self).setUp()
+    super(FileDepsTestBase, self).setUp()
     self.context(options={
```

Comment:
Let's delete this call to `self.context` which doesn't do anything, but did confuse us.